keyboards/handwired/not_so_minidox/keymaps/mtdjr/keymap.c

```diff
@@ -0,0 +1,207 @@
+#include "not_so_minidox.h"
```

Comment:
Please replace all of the includes  (except the solenoid one) with `#include QMK_KEYBOARD_H`